tools/cldr/cldr-to-icu/README.txt

```diff
@@ -1,104 +0,0 @@
-*********************************************************************
```

Comment:
I think rather than deleting this file, it might have been better to change it to just point to the .md file, since I suspect that there are links to this .txt file in various places.